go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -120,8 +114,7 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 
 		// Change our type back to the original value.
 		// Original type could be master so pass in a real value for masterTermStartTime
-		_, err = topotools.ChangeType(bgCtx, agent.TopoServer, tablet.Alias, originalType, tablet.Tablet.MasterTermStartTime)
-		if err != nil {
+		if err := agent.ChangeType(ctx, originalType); err != nil {
```

Comment:
The next call to refreshTablet is unnecessary, because ChangeType already calls it.